lib/spoom/sorbet/sigs.rb

```diff
@@ -34,7 +35,28 @@ def rbi_to_rbs(ruby_contents)
               sig.loc&.end_line&.pred,
               T.must(sig.loc).end_column,
             )
-            ruby_contents[start_index...end_index] = SigTranslator.translate(sig, node)
+            ruby_contents[start_index...end_index] = RBIToRBSTranslator.translate(sig, node)
+          end
+
+          ruby_contents
+        end
+
+        #: (String ruby_contents) -> String
+        def rbs_to_rbi(ruby_contents)
+          ruby_contents = ruby_contents.dup
+          rbs_comments = collect_rbs_comments(ruby_contents)
+
+          rbs_comments.each do |rbs_comment, node|
+            scanner = Scanner.new(ruby_contents)
+            start_index = scanner.find_char_position(
+              T.must(rbs_comment.loc&.begin_line&.pred),
+              T.must(rbs_comment.loc).begin_column,
+            )
+            end_index = scanner.find_char_position(
+              rbs_comment.loc&.end_line&.pred,
+              T.must(rbs_comment.loc).end_column,
+            )
+            ruby_contents[start_index...end_index] = RBSToRBITranslator.translate(rbs_comment, node)
```

Comment:
FYI I used `..` in tapioca. Can't remember exactly why but I think I was missing a new line otherwise. Tests here look good so I don't think we need it but I'm intrigued now.

Reply:
That's why I'm using `...` to exclude the end index which is the index of the `\n`, since I `.strip` the signature I'm creating.

Maybe you were not stripping the sig, meaning you had an extra new line after the sig and needed to remove the `\n` from the original one?